healthcare/internal/fhir-resource-conditional-delete/fhir_resource_conditional_delete.go

```diff
@@ -1,72 +0,0 @@
-// Copyright 2019 Google LLC
```

Comment:
Looks like this file is still being referenced in docs. Are we sure we want to delete it? The docs won't be publishable as-is if we merge this PR right now.

Reply:
Undeleted but left as v1beta1.